src/Data/Connection/ContentTypeConnectionResolver.php

```diff
@@ -18,20 +14,6 @@ class ContentTypeConnectionResolver extends AbstractConnectionResolver {
 	 */
 	protected $query;
 
-	/**
```

Comment:
This is useless code, but I'm totally happy to keep it if felt strongly. :)

Reply:
Nope. It's an oversight from #2457 and safe to remove